src/fragments/start/getting-started/next/api.mdx

```diff
@@ -224,9 +224,9 @@ Open **pages/index.js** and replace it with the following code:
 import { Authenticator } from '@aws-amplify/ui-react';
 import { Amplify, API, Auth, withSSRContext } from 'aws-amplify';
 import Head from 'next/head';
-import awsExports from '../src/aws-exports';
-import { createPost } from '../src/graphql/mutations';
-import { listPosts } from '../src/graphql/queries';
+import awsExports from '@/aws-exports';
```

Comment:
Confirmed that @/ points to the `src` folder

Reply:
This is accurate on old Next.js and new?